common/dtd/ldmlSupplemental.dtd

```diff
@@ -1225,6 +1225,25 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ATTLIST coverageLevel match CDATA #REQUIRED >
     <!--@MATCH:any-->
 
+<!ELEMENT pathGroups ( pathGroup+ ) >
```

Comment:
I think this can be simpler. All we need is an on-off switch.

Reply:
It is an on off. Trying to colocate with coverageLevels since it is a similar dataset. Let's discuss tomorrow.

Reply:
What I mean is, we just need the match attribute. We don't need the other flags(inLanguage, inScript/) etc. 

All we need to map an ID to is a set of paths.

Reply:
Simplest would be a single new element:

<pathMatch id=**NMTOKENS** match=**pathMatch, like coverageLevel**/>